frontend/src/my_team/MyTeamView.jsx

```diff
@@ -99,27 +110,42 @@ function MyTeamView() {
         </div>
       )}
 
-      {!displayScout && recommendations.data && (
-        <button onClick={handleScoutClick}> Scout Opponent</button>
+      {opponents.data && opponents.data.length == 0 && (
+        <Typography>
+          {" "}
+          Add players to opponent team to see scouting view
+        </Typography>
```

Comment:
<img width="1716" alt="Screenshot 2024-07-30 at 4 23 16 PM" src="https://github.com/user-attachments/assets/b814ca0b-d6eb-43a7-94bc-0b66a9ff1be1">
Prompts the user and disables the scout opponent button.